cos-event/run

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+# Env Vars:
+# REPOSITORY: name of the image registry/namespace to get the images
+# COS_ID: If set, specifies the full CRN of a Cloud Object Storage instance to use
+
+# Use a dynamic bucket name because COS prevents bucket name reuse for
+# 15 minutes after bucket deletion
+function bucket_name() {
+  set +x
+  PROJECT=$1
+  BASE=$(ibmcloud ce project get --name ${PROJECT}| awk '/^ID/{ print $2 }')
+  EPOCH=$(date +%s)
+  echo "${BASE}-${EPOCH}"
```

Comment:
I'd prefer if we didn't use a random string in the name because then we can't re-run this over and over when there are failures. The clean-up stuff won't delete the previous bucket if it failed.

Reply:
@duglin Okay, then you can only run it every 15 minutes (at most), because COS prevents bucket name reuse within that interval.  Not sure why, but they do.